sagemaker-experiments/tensorflow2-california-housing-regression-experiment/tensorflow2-california-housing-regression-experiment.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
<p>Where is the <code>fetch_california_housing()</code>  method defined?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Fixed to be more clear by using: `from sklearn.datasets import fetch_california_housing`